test/functional/test_framework/test_framework.py

```diff
@@ -169,7 +172,7 @@ def parse_args(self):
                             help="set a random seed for deterministically reproducing a previous test run")
         parser.add_argument("--descriptors", default=False, action="store_true",
                             help="Run test using a descriptor wallet")
-        parser.add_argument('--factor', type=float, default=1.0, help='adjust test timeouts by a factor')
+        parser.add_argument('--timeout-factor', dest="timeout_factor", type=float, default=1.0, help='adjust RPC timeout by a factor. Setting it to 0 disables the timeout')
```

Comment:
```suggestion
        parser.add_argument('--timeout-factor', dest="timeout_factor", type=float, default=1.0, help='adjust test timeouts by a factor. Setting it to 0 disables the timeout')
```

The previous wording was correct. This is not only about rpc timeouts

Reply:
Reverted.